docs/ref/contrib/gis/geos.txt

```diff
@@ -485,6 +485,21 @@ geometry is a point.
 Computes and returns a :class:`Point` guaranteed to be on the interior
 of this geometry.
 
+.. attribute:: GEOSGeometry.unary_union
+
+Computes the union of all the elements of this geometry.
+
+The result obeys the following contract:
+
+* Unioning a set of :class:`LineString`\ s has the effect of fully
+  noding and dissolving the linework.
+
+* Unioning a set of :class:`Polygon`\ s will always return a Polygonal
```

Comment:
should Polygonal be capitalized? I guess it might be meaning "a subclass of Polygon"?